samples/cf-multi-tenant-application/README.md

```diff
@@ -0,0 +1,54 @@
+# SAP Cloud SDK for JavaScript Multi-Tenant Sample Application
+
+## Description
+
+The code sample in this directory is a reference for the [multi-tenant application tutorial](https://sap.github.io/cloud-sdk/docs/js/tutorials/multi-tenant-application).
+You need to make adjustments to the code before deployment.
+The terms you need to replace are given in all caps and start with _`YOUR_`_ e.g. _`YOUR*REGION`*.
+
+Also, note that the samples here are intended as a didactic example and are not necessarily a best practice.
+The repositories' structure is as following:
+
+- [./multi-tenant-app](./multi-tenant-app) - Contains the code of the multi-tenant application.
+  It contains a simple service endpoint and logic to executed on subscription and unsubscription.
+- [./approuter](./approuter) - Contains the approuter necessary to attach JSON web tokens to the application.
+- [./service-config](./service-config) - Directory configurations for service instances.
+
+## Requirements
+
+The minimal requirements are:
+
+- A terminal to execute commands
```

Comment:
Do you mean CF CLI?

Reply:
yes but also the other commands like `npm i` or so will be needed most likely. I add the CF CLI as requirement.